tasks.py

```diff
@@ -43,6 +41,7 @@ def bin_prefix(cmd):
 @task
 def server(host=None, port=5000, debug=True, live=False):
     """Run the app server."""
+    from website import settings
```

Comment:
When website.settings is imported it requires kombu's Queue/Exchange classes which are dependencies of Celery. Moving the settings import into the tasks allows for traivs wheelhouse/requirements to be installed before other tasks attempt to import the settings module.